pkg/auth/providerrefresh/daemon_test.go

```diff
@@ -0,0 +1,49 @@
+package providerrefresh
+
+import (
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/robfig/cron"
+)
+
+func Test_updateRefreshCronTime(t *testing.T) {
+	ref := &refresher{}
+
+	ref.cron = *cron.New()
+	currentRefresh, err := cron.ParseStandard("0 0 * * *")
+	if err != nil {
+		t.Fatalf("error parsing cron time")
```

Comment:
Maybe include the error?

Generally, for "unexpected errors" I tend to use `t.Fatal(err)`

Reply:
sure, thanks for pointing it out